### PR TITLE
docs: allow image push to lima kubernetes cluster

### PR DESCRIPTION
**Pull request by afbjorklund**
### What issues does this PR fix or reference?

* #4486

### How to test this PR?

Testing requires:
* #4487

Most of these docs are identical between the Kubernetes providers.

Main difference is the thing about the ingress that is deployed on kind.

### CODE REVIEW

File: website/docs/kubernetes/lima/building-an-image-and-testing-it-in-lima.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+   1. Optionally, if your container is exposing more than one port, select the port to expose.
+   1. **Kubernetes namespaces**: select `default`.
+   1. Click **<icon icon="fa-solid fa-rocket" size="lg" /> Deploy**.
+   1. Wait for the pod to reach the state: **Phase: Running**.
+   1. Click **Done**.
+
+#### Verification
+
+1. The **<icon icon="fa-solid fa-cubes" size="lg" /> Pods** screen lists the running `my-image-container-pod` pod.
+1. Click on the pod name to view details and logs.
```
Comment (themr0c) [inline website/docs/kubernetes/lima/building-an-image-and-testing-it-in-lima.md]:
```suggestion
1. Click the pod name to view details and logs.
```

File: website/docs/kubernetes/lima/working-with-your-local-lima-cluster.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+#### Prerequisites
+
+- [You onboarded a Lima cluster](/docs/onboarding-for-kubernetes/lima).
+
+#### Procedure
+
+1. Open the Podman Desktop tray.
+2. Go to **Kubernetes**.
+3. Click on the Kubernetes context.
```
Comment (themr0c) [inline website/docs/kubernetes/lima/working-with-your-local-lima-cluster.md]: Another nitpick: 

```suggestion
3. Click the Kubernetes context.
```

**Review by themr0c — APPROVED** [review]
Nice!

**Comment by afbjorklund** [comment]
The "[vale](https://vale.sh/)" checker is not featured in the website directory, only in the toplevel github. So therefore it was not run...

```
E100 [loadStyles] Runtime error

style 'RedHat' does not exist on StylesPath

Execution stopped with code 1.
```

https://redhat-documentation.github.io/vale-at-red-hat/docs/main/user-guide/redhat-style-for-vale/

The language itself was copied from the "kind" documentation, so it should probably be fixed for both of them?

File: website/docs/kubernetes/lima/building-an-image-and-testing-it-in-lima.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+   1. Review the parameters that Podman Desktop has detected from your image definition.
+   1. Click **<icon icon="fa-solid fa-play" size="lg" /> Start Container** to start the container in your container engine.
+
+1. Test your image and container on your Lima cluster:
+
+   1. **<icon icon="fa-solid fa-cloud" size="lg" /> Search containers**: enter `my-custom-image-container` to find the running container.
+   1. Click **<icon icon="fa-solid fa-ellipsis-v" size="lg" /> > <icon icon="fa-solid fa-rocket" size="lg" /> Deploy to Kubernetes** to open the **Deploy generated pod to Kubernetes** screen.
+   1. **Pod Name**: keep the proposed value `my-custom-image-container-pod`.
+   1. Optionally, if your container is exposing more than one port, select the port to expose.
+   1. **Kubernetes namespaces**: select `default`.
```
Comment (jeffmaury) [inline website/docs/kubernetes/lima/building-an-image-and-testing-it-in-lima.md]:
```suggestion
   1. **Kubernetes Namespace**: select `default`.
```

File: website/docs/kubernetes/lima/building-an-image-and-testing-it-in-lima.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+   1. **Container name**: enter `my-custom-image-container`.
+   1. Review the parameters that Podman Desktop has detected from your image definition.
+   1. Click **<icon icon="fa-solid fa-play" size="lg" /> Start Container** to start the container in your container engine.
+
+1. Test your image and container on your Lima cluster:
+
+   1. **<icon icon="fa-solid fa-cloud" size="lg" /> Search containers**: enter `my-custom-image-container` to find the running container.
+   1. Click **<icon icon="fa-solid fa-ellipsis-v" size="lg" /> > <icon icon="fa-solid fa-rocket" size="lg" /> Deploy to Kubernetes** to open the **Deploy generated pod to Kubernetes** screen.
+   1. **Pod Name**: keep the proposed value `my-custom-image-container-pod`.
+   1. Optionally, if your container is exposing more than one port, select the port to expose.
```
Comment (jeffmaury) [inline website/docs/kubernetes/lima/building-an-image-and-testing-it-in-lima.md]: What about Ingress ?

File: website/docs/kubernetes/minikube/building-an-image-and-testing-it-in-minikube.md [hunk 5]
```diff
@@ -57,5 +57,5 @@ With Podman Desktop, you can build an image with your container engine, and test
 #### Verification
 
 1. The **<icon icon="fa-solid fa-cubes" size="lg" /> Pods** screen lists the running `my-image-container-pod` pod.
-1. Click on the pod name to view details and logs.
+1. Click the pod name to view details and logs.
```
Comment (jeffmaury) [inline website/docs/kubernetes/minikube/building-an-image-and-testing-it-in-minikube.md]: Should be on a separate PR

File: website/docs/kubernetes/minikube/working-with-your-local-minikube-cluster.md [hunk 6]
```diff
@@ -18,7 +18,7 @@ Set your Kubernetes context to your local Minikube-powered Kubernetes cluster.
 
 1. Open the Podman Desktop tray.
 2. Go to **Kubernetes**.
-3. Click on the Kubernetes context with the `minikube` name.
+3. Click the Kubernetes context with the `minikube` name.
```
Comment (jeffmaury) [inline website/docs/kubernetes/minikube/working-with-your-local-minikube-cluster.md]: Idem

File: website/docs/kubernetes/lima/building-an-image-and-testing-it-in-lima.md (hunk 4 above)
Comment (afbjorklund) [reply website/docs/kubernetes/lima/building-an-image-and-testing-it-in-lima.md]: There is no ingress deployed by default, only tunneled ports (like from NodePort)

**Comment by afbjorklund** [comment]
The whole section is gone now, but I guess it would end up in a special submenu ?

Like: https://podman-desktop.io/docs/kubernetes/kind/pushing-an-image-to-kind

File: website/docs/kubernetes/lima/building-an-image-and-testing-it-in-lima.md (hunk 3 above)
Comment (afbjorklund) [reply website/docs/kubernetes/lima/building-an-image-and-testing-it-in-lima.md]: Copied from https://podman-desktop.io/docs/kind/building-an-image-and-testing-it-in-kind

**Review by cdrage — APPROVED** [review]

**Review by benoitf — APPROVED** [review]

**Comment by slemeur** [comment]
Thanks for the PR @afbjorklund
